### PR TITLE
KOGITO-6603: Rename: `kogito-tooling` -> `kie-tools`

### PR DESCRIPTION
**Pull request by caponetto**
Workflows will fail since all references point to the new name.
For that reason, I conducted some tests on this temporary repo [1].

[1] https://github.com/caponetto/kie-tools/actions

### CODE REVIEW

File: .ci/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -50,16 +50,16 @@ pipeline {
...
                 expression { isNormalPRCheck() && isSonarCloudEnabled() }
             }
             steps {
                 script {
                     if (isSonarCloudEnabled()) {
-                        getMavenCommand('kogito-tooling/packages/stunner-editors')
+                        getMavenCommand('kie-tools/packages/stunner-editors')
                                 .withOptions(['-e', '-nsu', "-pl '!kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime,!kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-testing,!lienzo-webapp'"])
-                                .withProperty('sonar.projectKey', 'org.kie.kogito:kogito-tooling')
+                                .withProperty('sonar.projectKey', 'org.kie.kogito:kie-tools')
```
Comment (jomarko) [inline .ci/jenkins/Jenkinsfile]: We need to create this project on sonarcloud. Furthermore it needs to have same secrets (especially key, value may differ) as we have currently for `org.kie.kogito:kogito-tooling`

Screenshot from 2022-01-26 08-58-47

File: .github/workflows/sonar_analysis_stunner_editors.yml [hunk 2]
```diff
@@ -50,4 +50,4 @@ jobs:
           SONARCLOUD_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: |
           cd packages/stunner-editors
-          mvn -B validate org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Dsonar.projectKey=org.kie.kogito:kogito-tooling -Dsonar.organization=kiegroup -Dsonar.host.url=https://sonarcloud.io -DskipTests -Dgwt.compiler.skip -Dsonar.login=${{ env.SONARCLOUD_TOKEN }} -pl '!kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime,!kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-testing,!lienzo-webapp'
+          mvn -B validate org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Dsonar.projectKey=org.kie.kogito:kie-tools -Dsonar.organization=kiegroup -Dsonar.host.url=https://sonarcloud.io -DskipTests -Dgwt.compiler.skip -Dsonar.login=${{ env.SONARCLOUD_TOKEN }} -pl '!kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime,!kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-testing,!lienzo-webapp'
```
Comment (jomarko) [inline .github/workflows/sonar_analysis_stunner_editors.yml]: comment above affects also this place.

**Review by jomarko — CHANGES_REQUESTED** [review]
Thank you for the PR. I have a comment about the sonar.

File: .ci/jenkins/Jenkinsfile (hunk 1 above)
Comment (tiagobento) [reply .ci/jenkins/Jenkinsfile]: @jomarko Please go ahead and create the necessary infra you need on Sonar.

File: .github/workflows/sonar_analysis_stunner_editors.yml (hunk 2 above)
Comment (tiagobento) [reply .github/workflows/sonar_analysis_stunner_editors.yml]: Same as above.

**Review by tomasdavidorg — APPROVED** [review]
Looks good to me.

File: .github/workflows/sonar_analysis_stunner_editors.yml (hunk 2 above)
Comment (jomarko) [reply .github/workflows/sonar_analysis_stunner_editors.yml]: @tiagobento I do not have permission to do it in kiegroup.

Comment (tiagobento) [reply .github/workflows/sonar_analysis_stunner_editors.yml]: @jomarko Do you know who has? If so, please contact them to properly set it up.

Comment (jomarko) [reply .github/workflows/sonar_analysis_stunner_editors.yml]: @tiagobento I sent an email to bsig. I do not know. Lets wait for responses there.

**Comment by tiagobento** [comment]
@caponetto @jomarko Caponetto, can you please revert the changes on the Sonar organization? We can keep using the old one until we have a new one. This will unblock the merge.

**Comment by caponetto** [comment]
@tiagobento Sure, I'll do it!

**Comment by caponetto** [comment]
Done in https://github.com/kiegroup/kogito-tooling/pull/791/commits/37f5fd5d71b16fcf5fb85a73dd8bc198388e76ee.
@tiagobento @jomarko

**Review by jomarko — APPROVED** [review]

**Comment by jomarko** [comment]
Thank you @caponetto

**Review by tiagobento — APPROVED** [review]

**Pull request merged**